src/placeos-driver/proxy/system.cr

```diff
@@ -60,7 +58,7 @@ class PlaceOS::Driver::Proxy::System
     index = index.to_i
 
     module_id = @system["#{module_name}/#{index}"]?
-    metadata = @mutex.synchronize { @redis.get("interface/#{module_id}") } if module_id
+    metadata = RedisStorage.get("interface/#{module_id}") if module_id
     metadata = if module_id && metadata
```

Comment:
redundant module_id check on this line

Reply:
but `module_id` could be `nil` as the line above has `]?`